tests/test_wsi_registration.py

```diff
@@ -6,9 +6,12 @@
 
 from tiatoolbox.tools.registration.wsi_registration import (
     DFBRegister,
+    apply_bspline_transform,
```

Comment:
Please can you do a spell check on "initializer"?

https://github.com/TissueImageAnalytics/tiatoolbox/blob/7d40d88ca4c46b4ccbdbeefa31b1abf0c4e76b92/tests/test_wsi_registration.py#L310

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/1760d595678752b932649eaebbf677d9814bdd5a/tests/test_wsi_registration.py#L307-L310